src/test/java/org/example/AppTest.java

```diff
@@ -1,38 +1,48 @@
 package org.example;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
-
-/**
- * Unit test for simple App.
- */
-public class AppTest 
-    extends TestCase
-{
-    /**
-     * Create the test case
-     *
-     * @param testName name of the test case
-     */
-    public AppTest( String testName )
-    {
-        super( testName );
-    }
+import org.assertj.core.data.Index;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AppTest {
 
-    /**
-     * @return the suite of tests being tested
-     */
-    public static Test suite()
-    {
-        return new TestSuite( AppTest.class );
+    @Test
+    void testCarsSortedByYear(){
+        Car toyota = new Car(1990,2.4);
+        Car mazda = new Car(2006,1.8);
+        Car opel = new Car(1989,2.5);
+        Car acura = new Car(2019,1.6);
+
+        List<Car> carsList = Arrays.asList(toyota, mazda, opel, acura);
+        Collections.sort(carsList);
+
+        assertThat(carsList.size()).isEqualTo(4);
+        assertThat(carsList).isNotEmpty()
+                .contains(toyota, mazda, opel, acura)
```

Comment:
containsExactly() would do the job :) 

https://medium.com/enfuse-io/testing-java-collections-with-assertj-2fa678d80952